composables/projectProperties.ts

```diff
@@ -28,10 +32,21 @@ export function useProjectProperties() {
    */
   async function fetchProjectProperties(locale: string) {
     try {
-      return (
-        cachedProjectProperties.value[locale] ||
-        (await $fsxaApi.fetchProjectProperties({ locale }))
-      )
+      const projectPropertiesResponse = await fsxaApi.fetchProjectProperties({
+        locale
+      })
+      let projectProperties = null
+      if (
+        projectPropertiesResponse &&
+        Object.hasOwn(projectPropertiesResponse as Object, 'projectProperty')
+      ) {
+        projectProperties = (
+          projectPropertiesResponse as NormalizedProjectPropertyResponse
+        ).projectProperties
+      }
+      projectProperties = projectPropertiesResponse as ProjectProperties
+
+      return cachedProjectProperties.value[locale] || projectProperties
```

Comment:
I think this is a bug here ->
We fetch project properties without checking the cache before